stdlib/importlib/metadata/__init__.pyi

```diff
@@ -241,7 +241,6 @@ class MetadataPathFinder(DistributionFinder):
     @classmethod
     def find_distributions(cls, context: DistributionFinder.Context = ...) -> Iterable[PathDistribution]: ...
     if sys.version_info >= (3, 10):
-        # Yes, this is an instance method that has a parameter named "cls"
```

Comment:
Added in #6443

Reply:
Don't feel strongly, but this one is kind of funny and might help a human who is quickly glancing at stubs

Reply:
I have no issue leaving it in if you feel it has value outside of explaining a `type: ignore` comment